web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -51,101 +59,231 @@ const useStyles = makeStyles({
-                          return (
-                            <React.Fragment key={`zone-${areaName}`}>
-                              <TableRow
-                                className={classes.planningArea}
-                                key={`zone-${areaName}`}
-                              >
-                                <TableCell className={classes.planningArea} colSpan={3}>
-                                  {area.name}
-                                </TableCell>
-                              </TableRow>
-                              {Object.entries(area.stations).map(
-                                ([stationCode, station]) => {
-                                  return (
-                                    <TableRow
-                                      className={classes.station}
-                                      key={`station-${stationCode}`}
-                                    >
-                                      <TableCell key={`station-${station.code}-name`}>
-                                        {station.station_props.name} ({station.code})
-                                      </TableCell>
-                                      <TableCell
-                                        key={`station-${station.code}-elevation`}
-                                      >
-                                        {station.station_props.elevation}
-                                      </TableCell>
-                                      <TableCell
-                                        key={`station-${station.code}-fuel-type`}
-                                      >
-                                        {station.station_props.fuel_type.abbrev}
-                                      </TableCell>
-                                    </TableRow>
-                                  )
-                                }
-                              )}
-                            </React.Fragment>
-                          )
-                        })}
-                      </React.Fragment>
-                    )
-                  })}
-                </TableBody>
-              </Table>
-            </TableContainer>
-          </Paper>
-        </div>
-      )}
+      <div className={classes.controls}>
+        <Typography component="div" variant="subtitle2">
+          {props.title}
+        </Typography>
+        <Button color="primary" onClick={props.previousDay}>
+          Previous
+        </Button>
+        {props.currentDay}
+        <Button color="primary" onClick={props.nextDay}>
+          Next
+        </Button>
+      </div>
+      <Paper className={classes.paper} elevation={1}>
+        <TableContainer className={classes.tableContainer}>
+          <Table stickyHeader aria-label="daily table view of HFI by planning area">
+            <TableHead>
+              <TableRow>
+                {/* TableHead and TableRow don't apply classes.tableHeader styling - has to be assigned to TableCell */}
```

Comment:
It does create th cells with the class 'MuiTableCell-head' - so you can apply that in your tableHeader class. (see above)

Reply:
Artifact from: https://github.com/bcgov/wps/pull/1045

But can address.